src/sbt-test/sbt-1.5/testkit/build.sbt

```diff
@@ -0,0 +1,78 @@
+lazy val V = _root_.scalafix.sbt.BuildInfo
+
+lazy val rulesCrossVersions = Seq(V.scala213, V.scala212, V.scala211)
+lazy val scala3Version = "3.0.0"
+
+inThisBuild(
+  List(
+    semanticdbEnabled := true,
+    semanticdbVersion := scalafixSemanticdb.revision
+  )
+)
+
+lazy val rules = projectMatrix
+  .settings(
+    libraryDependencies += "ch.epfl.scala" %% "scalafix-core" % V.scalafixVersion
+  )
+  .defaultAxes(VirtualAxis.jvm)
+  .jvmPlatform(rulesCrossVersions)
+
+lazy val input = projectMatrix
+  .defaultAxes(VirtualAxis.jvm)
+  .jvmPlatform(
+    scalaVersions = rulesCrossVersions,
+    settings = Seq(
+      scalacOptions += "-P:semanticdb:synthetics:on"
+    )
+  )
+  .jvmPlatform(scalaVersions = Seq(scala3Version))
+
+lazy val output = projectMatrix
+  .defaultAxes(VirtualAxis.jvm)
+  .jvmPlatform(scalaVersions = rulesCrossVersions :+ scala3Version)
+
+lazy val testsAggregate = Project("tests", file("tests/aggregate"))
```

Comment:
arbitrary path to avoid recompiling, we just want this for `tests/test` to aggregate over the `tests*`